src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -137,6 +137,13 @@ def load_arguments(self, _):
         validator=validate_tag_filters
     )
 
+    kv_revision_retention_days_arg_type = CLIArgumentType(
+        options_list=['--kv-revision-retention-days'],
+        type=int,
+        help='Number of days to retain revisions of key-values in the App Configuration store. Defaults to 7 days for Free and Developer SKU stores and 30 days for Standard and Premium SKU stores',
```

Comment:
I know we use `sku` and `tier` interchangeably. When we use `sku` it seems to be lower case. Perhaps, we can update this to match other instances.

I hope we can one day to a cleanup to use one term consistently🙂